docker/py/Dockerfile

```diff
@@ -29,7 +29,6 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     python-dev \
     unzip \
     vim && \
-    apt-get clean && \
```

Comment:
@rokroskar why do you remove this command here? Isn't it good to purge the cache?

Reply:
indeed, thanks for catching that... a cut-and-paste error :)